core/logger/redirectStderrWindows.go

```diff
@@ -0,0 +1,35 @@
+//+build windows
+
+package logger
+
+import (
+	"os"
+	"syscall"
+)
+
+var (
+	kernel32         = syscall.MustLoadDLL("kernel32.dll")
+	procSetStdHandle = kernel32.MustFindProc("SetStdHandle")
+)
+
+func setStdHandle(stdhandle int32, handle syscall.Handle) error {
+	r0, _, e1 := syscall.Syscall(procSetStdHandle.Addr(), 2, uintptr(stdhandle), uintptr(handle), 0)
+	if r0 == 0 {
+		if e1 != 0 {
+			return error(e1)
+		}
+		return syscall.EINVAL
+	}
+
+	return nil
+}
+
+// redirectStderr to the file passed in
+func RedirectStderr(f *os.File) error {
```

Comment:
I think this should not be exported, to match the linux version, right?

Reply:
Right, missed that one. Changed.